application/src/main/java/com/sanction/thunder/authentication/Key.java

```diff
@@ -0,0 +1,49 @@
+package com.sanction.thunder.authentication;
+
+import com.google.common.base.MoreObjects;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class Key {
+
+  private final String application;
+  private final String secret;
+
+  public Key(String application, String secret) {
+    this.application = checkNotNull(application);
+    this.secret = checkNotNull(secret);
+  }
+
+  public String getApplication() {
+    return application;
+  }
+
+  public String getSecret() {
+    return secret;
+  }
+
+  /**
+   * Compares for equality in another key object.
+   *
+   * @param key another object of type Key
+   * @return a boolean
+   */
+  public boolean equals(Key key) {
+    if (!this.application.equals(key.getApplication())) {
+      return false;
+    }
+
+    if (!this.secret.equals(key.getSecret())) {
+      return false;
+    }
+
+    return true;
+  }
```

Comment:
Can you make this method look similar to the `equals()` method in `StormUser.java`?